doc/manual/manual.tex

```diff
@@ -10818,6 +10818,125 @@ \subsubsection{Yamauchi \& Takei anelastic shear wave velocity-temperature conve
+
+The parameter file is available at \url{benchmarks/differenciation/diff_adim.prm}. 
+
+\paragraph{First time step}
+
+The first time step solves the velocity equation for a constant porosity. 
+The 1-D equations with constant porosity $\phi_0$ have an analytical solution for the matrix velocity given by  \cite{SRPHD07} as
+\begin{equation}
+    v_m(y) = -\delta ^2 \phi_0^2 (1-\phi_0)\left [ 1+\frac{\sinh\left ( \frac{1}{h}\right )\sinh\left (\frac{y}{h}\right )}{\cosh\left ( \frac{1}{h}\right ) +1} - \cosh\left ( \frac{y}{h}\right )\right ], 
+\end{equation}
+where $h$ is length scale related to the compaction length as 
+\begin{equation}
+    h=\delta \sqrt{\phi_0 \left (1+\frac{4}{3}\phi_0 \right )(1-\phi_0)}.
+\end{equation}
+
+Comparison between the analytical solution and our runs for two compaction lengths (1 and 0.2) are shown on figure \ref{fig:analytic}. 
+The agreement is good, with a slight discrepancy at the top, likely due to the top boundary condition not exactly defines at $y=1$.   
+
+\begin{figure}[h]
+    \centering
+    \includegraphics[width=0.49\textwidth]{cookbooks/benchmarks/delta_1_analytical.pdf}
+    \hfill
+    \includegraphics[width=0.49\textwidth]{cookbooks/benchmarks/delta_02_analytical.pdf}
+    \caption{Comparison between analytical solution (yellow x symbols) and velocity of the matrix at the first time step (blue solid line), for a compaction length of 1 (left figure) and 0.2 (right figure).}
+    \label{fig:analytic}
+\end{figure}
+
+\paragraph{Evolution}
+
+The porosity profiles evolves such that the liquid is expelled to the top and the matrix compacts at the bottom. 
+No flow can cross the top or bottom boundaries. 
+We first check the mass conservation of the liquid with time, for different number of points in the domain. 
+Results are shown on figure \ref{fig:comp}. 
+There is an overshoot at the beginning, likely due to the formation of the wiggles, and then depending on the number of points used, the system tends to stabilize.  
+Results seem to show that at least 200 points are required for this case of $\delta = 1$. 
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.5\textwidth]{cookbooks/benchmarks/comparison_pts.pdf}
+    \caption{Evolution of the total mass of liquid (normalized by the initial mass) for different discretization, with a compaction length $\delta=1$. }
+    \label{fig:comp}
+\end{figure}
+
+
+The porosity and matrix velocity evolution are shown on figure \ref{fig:evolution}. 
+Results are similar to \cite{SRPHD07}, except for porosity close to 1, where \aspect{} do not converge correctly and overestimate the segregation. 
+Since the model was designed to solve for very low porosity, and that the equations should not be physically valid for porosity high enough than the matrix is no longer connected (the highest threshold usually considered is about 40\%, corresponding to a system made of spheres with a random close packing)
```

Comment:
This sentence is not complete. You likely want to say the above deviation is expected, or at least explainable?

Reply:
modified